src/plugins/index_pattern_field_editor/public/components/preview/field_list/field_list_item.tsx

```diff
@@ -31,12 +36,42 @@ export const PreviewListItem: React.FC<Props> = ({ field: { key, value }, highli
         <div className="indexPatternFieldEditor__previewFieldList__item__key__wrapper">{key}</div>
       </EuiFlexItem>
       <EuiFlexItem className="indexPatternFieldEditor__previewFieldList__item__value">
-        <EuiToolTip position="top" content={value}>
-          <span className="indexPatternFieldEditor__previewFieldList__item__value__wrapper">
-            {value}
-          </span>
+        <EuiToolTip
+          position="top"
+          content={typeof value !== 'string' ? JSON.stringify(value) : value}
+        >
+          {formattedValue !== undefined ? (
+            <span
+              className="indexPatternFieldEditor__previewFieldList__item__value__wrapper"
+              // We  can dangerously set HTML here because this content is guaranteed to have been run through a valid field formatter first.
+              dangerouslySetInnerHTML={{ __html: formattedValue! }} // eslint-disable-line react/no-danger
```

Comment:
For things like the image url formatter, do we want to show a preview? As is, it doesn't scale well within the table. Perhaps we could show an icon or something that shows the full image on click/hover?

![kibana_sample_data_logs-Elastic](https://user-images.githubusercontent.com/5226211/123272786-1412e200-d4d0-11eb-9f21-dd6715828e66.png)

![kibana_sample_data_logs-Elastic2](https://user-images.githubusercontent.com/5226211/123272789-14ab7880-d4d0-11eb-99a8-f4838eb81747.png)